tools/src/test/scala/caliban/tools/SchemaWriterSpec.scala

```diff
@@ -174,9 +183,6 @@ object SchemaWriterSpec extends DefaultRunnableSpec {
           )
         )
       },
-      testM("empty schema test") {
```

Comment:
Any reason to remove that one?

Reply:
I removed this test, because on different OS line separators differs, so it's failed on windows. Also I find this test not so useful, but if you want I can return it with corrected line separator

Reply:
Fine with me 👍

Reply:
@ghostdogpr I returned corrected test